### PR TITLE
Align to the code style

### PR DESCRIPTION
**Pull request by suda**

### CODE REVIEW

**Comment by dmiddlecamp** [comment]
:-1: cannot change the output of the module

**Comment by dmiddlecamp** [comment]
:-1: also don't change the interface of the module (function names, etc), this module is in use.

**Comment by suda** [comment]
@dmiddlecamp this is why I bumped minor version. It's only used in buildpacks which have `0.0.1` pinned to them.

File: cli.js [hunk 1]
```diff
@@ -24,12 +24,12 @@ var p = new HalModuleParser();
...
 	process.exit(-1);
 }
 var filename = process.argv[2];
 
 
 p.parseFile(filename, function() {
-	console.log("got ", arguments);
-});
\ No newline at end of file
+	console.log('got', arguments);
```
Comment (dmiddlecamp) [inline cli.js]: leave space in here?

File: lib/HalDependencyResolver.js [hunk 2]
```diff
@@ -227,24 +225,24 @@ HalDependencyResolver.prototype = {
 		var results = [];
 
 		// expecting something like...
-		// { f: "s", n: "2", v: 2 }
+		// { f: 's', n: '2', v: 2 }
```
Comment (dmiddlecamp) [inline lib/HalDependencyResolver.js]: double-quotes in JSON

File: lib/HalModuleParser.js [hunk 3]
```diff
@@ -210,26 +215,38 @@ HalModuleParser.prototype = {
...
+	 *	platformID: number,
+	 *	moduleFunction: number,
+	 *	moduleIndex: number,
+	 *	depModuleFunction: number,
+	 *	depModuleIndex: number,
+	 *	depModuleVersion: number
+	 * }
 	 * @param fileBuffer
-	 * @returns {{moduleStartAddy: string, moduleEndAddy: string, moduleVersion: number, platformID: number, moduleFunction: number, moduleIndex: number, depModuleFunction: number, depModuleIndex: number, depModuleVersion: number}}
+	 * @returns Object
```
Comment (dmiddlecamp) [inline lib/HalModuleParser.js]: leave old returns statement?

File: lib/utilities.js [hunk 4]
```diff
@@ -39,12 +39,12 @@ module.exports = {
...
 		}
 
-		var same = (left.length == right.length),
+		var same = (left.length === right.length),
 			i = 0,
 			max = left.length;
 
 		while (i < max) {
-			same &= (left[i] == right[i]);
+			same &= (left[i] === right[i]);
```
Comment (dmiddlecamp) [inline lib/utilities.js]: this should be "=="s maybe?

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "binary-version-reader",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (dmiddlecamp) [inline package.json]: oh!  let's start a changelog?

**Comment by dmiddlecamp** [comment]
left some really minor comments, otherwise, awesome, thanks!

File: cli.js (hunk 1 above)
Comment (suda) [inline cli.js]: Space is automatically added if you separate arguments instead merging them into one string:

**Pull request merged**